sdcm/group_common_events.py

```diff
@@ -51,3 +51,9 @@ def ignore_no_space_errors(node):
             DbEventsFilter(type='DATABASE_ERROR', line='No space left on device', node=node), \
             DbEventsFilter(type='FILESYSTEM_ERROR', line='No space left on device', node=node):
         yield
+
+
+@contextmanager
+def ignore_ycsv_connection_refused():
```

Comment:
s/ycsv/ycsb/

Reply:
you are saying you fixed this... but it's still here

Reply:
Fixd